packages/notifications-panel/CHANGELOG.md

```diff
@@ -0,0 +1,282 @@
+# [@hig/notifications-flyout-v3.2.0](https://github.com/Autodesk/hig/compare/@hig/notifications-flyout@3.1.1...@hig/notifications-flyout@3.2.0) (2022-06-15)
```

Comment:
I think we want to clear this log to just say v0.0.1 of notification-panel, and it leverages  @hig/notifications-flyout@3.2.0 plus adding `Mark All As Read Feature`. We dont have to inherit their log